src/core/public/fatal_errors/fatal_errors_service.test.ts

```diff
@@ -100,9 +100,7 @@ describe('start.add()', () => {
   it('exposes a function that passes its two arguments to fatalErrors.add()', () => {
     const { fatalErrors, i18n } = setup();
 
-    jest.spyOn(fatalErrors, 'add').mockImplementation(() => {
-      /* noop */
-    });
+    jest.spyOn(fatalErrors, 'add').mockImplementation(() => undefined as never);
```

Comment:
Doesn't the default mock implementation return `undefined`?

Reply:
not really. `add` [throws an error and does't return anything](https://github.com/elastic/kibana/blob/master/src/core/public/fatal_errors/fatal_errors_service.tsx#L69), so it's signature is
```js
(error: Error | string, source?: string) => never
```

Reply:
Ah got it. I forgot that spies call the real implementation by default.